docs/faq.md

```diff
@@ -1,36 +1,41 @@
 # FAQ
 
 These are frequently asked questions. If you have any questions
-  not on this list, then **please** open an issue and ask.
-  
+not on this list, then **please** [open an issue and ask][new-issue].
+
+[new-issue]: https://github.com/Raynos/mercury/issues/new
+
+
```

Comment:
An uncommon style perhaps. Find that it breaks up the sections better visually in markdown files. I'll remove the extra line though because the rest of the faq.md follow the single line between sections.